ocaml/xapi/suite.ml

```diff
@@ -69,6 +69,7 @@ let base_suite =
     Test_network_event_loop.test;
     Test_network.test;
     Test_pusb.test;
+    Test_host.test;
```

Comment:
Should we use alcotest for new tests (suite_alcotest.ml)?

Reply:
Hmm, I think you might be right.

Reply:
API doc is here: https://mirage.github.io/alcotest/alcotest/Alcotest/index.html
And here are some examples:
* https://github.com/xapi-project/xen-api/pull/3421
* https://github.com/xapi-project/xen-api/pull/3420